src/System.Runtime.TimeZoneData/System.Runtime.TestData.csproj

```diff
@@ -0,0 +1,34 @@
+<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+      <TimeZoneDataVersion>2020a</TimeZoneDataVersion>
+	  <TimeZoneSource>https://data.iana.org/time-zones/tzdb</TimeZoneSource>
+	  <InputFolder>$(MSBuildProjectDirectory)/input</InputFolder>
+	  <OutputFolder>$(MSBuildProjectDirectory)/zoneinfo</OutputFolder>
+    </PropertyGroup>
+	<ItemGroup>
+		<Area Include="africa" />
+		<Area Include="antarctica" />
+		<Area Include="asia" />
+		<Area Include="australasia" />
+		<Area Include="etcetera" />
+		<Area Include="europe" />
+		<Area Include="northamerica" />
+		<Area Include="southamerica" />
+	</ItemGroup>
+	<ItemGroup>
+	</ItemGroup>
+	<Target Name="DownloadData" BeforeTargets="Build" >
```

Comment:
should we check in the downloaded timezone data files instead of doing a fresh download on each build?
this would map it to how System.Private.Runtime.UnicodeData does it.